src/action.ts

```diff
@@ -1,6 +1,5 @@
 import * as core from "@actions/core";
```

Comment:
can we please remove unused imports?

Reply:
Fixed this via lint introduction. I'll make a separate PR adding a github action to enforce lint.